src/pages/registration/references_tab/components/PublisherField.tsx

```diff
@@ -21,7 +21,12 @@ const PublisherField: FC = () => {
           placeholder={t('references.search_for_publisher')}
           errorFieldName={ReferenceFieldNames.PUBLICATION_CONTEXT_PUBLISHER}
           setValue={(newValue) => {
-            const contextValues = formatPublicationContextWithPublisher(value.type, newValue);
+            const contextValues: any = formatPublicationContextWithPublisher(value.type, newValue);
+            if (newValue && value.seriesTitle) {
```

Comment:
tror vi må finne en bedre løsning på å håndtere publisher, journal og series enn hva som er nå. Feks at vi lagrer hele objektet for for både publisher og serie i frontend (formik), så slår vi de sammen når vi lagrer.
Hvis man har valgt både publisher og serie skal NPI-nivå til serie gjelde, men hvis man fjerner serie nå så har vi ikke lenger tilgang til nivå for publisher som nå skal gjelde, så det er noen bugs her atm :/

det var uansett mer feil før, så tar det ikke i denne runden.